toolkits/arcade_expert/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "arcade_arcade_expert"
```

Comment:
Given that the toolkit name starts with `arcade` and official Arcade toolkits start with `arcade`, we end up with `arcade_arcade_expert`. If we were to change it to `arcade_expert`, then the toolkit name would be `Expert` instead of `ArcadeExpert`.

Pip installing:
`pip install arcade_arcade_expert`

Calling an ArcadeExpert tool:
`arcade_client.tools.execute("ArcadeExpert.SearchDocumentation", ...)`